src/components/molecules/CartCard/CartCard.tsx

```diff
@@ -1,3 +1,225 @@
+  description?: ReactNode;
+  discount?: Discount;
+}
+
+const StyledCardCard = styled.div`
+  .title {
+    font-size: 30px;
+    margin-bottom: 20px;
+    font-weight: 700;
+  }
+  border-radius: ${(props) => props.theme.cardRadius}px;
+  background: ${(props) =>
+    props.theme.mode !== "dark" ? props.theme.white : props.theme.gray1};
+  padding: 40px;
+  .buy-button {
+    margin-bottom: 23px;
+  }
+  .separator {
+    height: 1px;
+    width: 24px;
+    background: ${(props) =>
+      props.theme.mode !== "dark" ? props.theme.gray1 : props.theme.white};
+    margin: 20px 0 14px 0;
+  }
+  .open-discount-state-container {
+    margin-left: 24px;
+  }
+  .discount-form-container {
+    margin-top: 17px;
+    display: flex;
+    flex-direction: column;
+    gap: 28px;
+  }
+  .discount-toggle {
+    display: inline-block;
+    padding: 6px 0;
+    cursor: pointer;
+  }
+  .discount-granted-info {
+    margin-bottom: 23px;
+    font-size: 14px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    white-space: pre-wrap;
+  }
+  .discount-remove {
+    display: flex;
+    margin-left: 12px;
+    cursor: pointer;
+  }
+  .granted-info {
+    margin-left: 12px;
+    color: #27ae60;
+    font-size: 12px;
+  }
+`;
+
+export const CartCard: React.FC<CartCardProps> = (props) => {
+  const { id, title, subtitle, onBuyClick, description, discount } = props;
+  const { t } = useTranslation();
+
+  const initialDiscountOpen = useMemo(() => {
+    if (discount) {
+      return discount.isOpen ? discount.isOpen : false;
+    }
+    return false;
+  }, [discount]);
+
+  const [isDiscountOpen, setIsDiscountOpen] =
+    useState<boolean>(initialDiscountOpen);
+  const [discountInput, setDiscountInput] = useState<string>("");
+
+  return (
+    <StyledCardCard>
+      <Text className="title">{title}</Text>
+      {subtitle}
+      <Button
+        mode="secondary"
+        block
+        className="buy-button"
+        onClick={() => onBuyClick(id)}
+      >
+        {t("cart.card.buy.button")}
+      </Button>
+      {discount && (
+        <>
+          {discount.grantedDiscountCodes.map((code: string) => {
+            return (
+              <Text className="discount-granted-info">
+                <ReactMarkdown components={{ p: "span" }}>
+                  {t("cart.card.discount.granted", { code })}
+                </ReactMarkdown>
+                <span
+                  className={"discount-remove"}
+                  onClick={() => discount.onDeleteDiscoundClick(id, code)}
+                >
+                  <IconBin />
+                </span>
+              </Text>
+            );
+          })}
+        </>
+      )}
+      {description}
+      {discount && (
+        <>
+          <div className="separator"></div>
+          <div
+            className="discount-toggle"
+            onClick={() => setIsDiscountOpen(!isDiscountOpen)}
+          >
+            {t("cart.card.add.discount.button")}
```

Comment:
just use prefix for component then cammelCase 

```suggestion
            {t("cart.cardAddDiscountButton")}
```